contrib/config_regtest.toml

```diff
@@ -7,71 +7,35 @@ data_dir = "/path/to/your/datadir/revault"
-	"xpub6FEZyiJxqwu7zkqqVGXVbGhcAj1L5imn55VVa4Mk5WE46xdAKgD4uSR9ems9EehAApZPVXFrxcLQ7zPqYywu1z4Cjhesyp7HeRSgSdUq1BB",
-	# xprv9yFvjFWhx6hEMcPi2JASzKSVS1mszhDKWBE4TxfQxDUkJrTFM7h1gQnBTXEqEUP2fB98hwU6fYoA6UUmkP2PAeHMUEe94uHr9SD5T72LdYe
-	"xpub6CFH8m3bnUFXa6UB8KhTMTPDz3cNQ9wAsQ9fGM52WZ1jBenPtf1GED6fJoDmpEYQQkk3VUHFN5ZRDLV7SRgX4M8KMTpRTH9zGRzg5udqwwo",
-	# xprvA1UEBJLZFm1oZUKLubtgeqhYBKjKPQqWaw5qMkg7GyyEjc9PQs5HiBVR3CpEAAawkF9jhd9LyV6arxBCYat8DAF5ChDDDKYB4GCVwUGz87C
-	"xpub6ETaaosT68a6mxPp1dRh1yeGjMZonsZMxA1SA95iqKWDcQUXxQPYFyottUR58E8qjjnAwPcEtYS9iejkERbnGuNqfF2wgToLcxzf97FHevs"
-]
-
-# The keys of all the cosigners
-# This option MUST NOT be changed after running revaultd for the first time. If you want to change it, please remove the following files:
-# - /path/to/your/data_dir/revaultd.sqlite3
-# - /path/to/your/data_dir/revaultd-watchonly-wallet-1
-cosigners_keys = [
-	"02644cf9e2b78feb0a751e50502f530a4cbd0bbda3020779605391e71654dd66c2",
-	"03ced55d1208bd8c6b42b11e29baa577711cae831b3a1296607c5e5d3ed365f49c",
-	"026237f655f3bf45fd6b7aa00e91c2603d6155f1cc001e40f5e47662d965c4c779",
-	"030a3cbcfbfdf7122fe7fa830354c956ea6595f2dbde23286f03bc1ec0c1685ca3",
-]
-
-# The xpubs of all the managers
-# This option MUST NOT be changed after running revaultd for the first time. If you want to change it, please remove the following files:
-# - /path/to/your/data_dir/revaultd.sqlite3
-# - /path/to/your/data_dir/revaultd-watchonly-wallet-1
-managers_xpubs = [
-	# xprv9yFvjFWhx6hEhx2fRYSC31pU3nH3Mrok4sBwUt4pV3RJZYN1oex2imuuxUt3eyhbZ2mKdBAWQaVyhg3WXuzrnRhRZbicr2X5hmNuhTxHGEn
-	"xpub6CFH8m3bnUFXvS78XZyCQ9mCbp7XmKXbS67YHGUS3NxHSLhAMCGHGaEPojcoYt5PYnocyuScAM5xuDzf4BqFQt3fhmKEaRgmVzDcAR46Byh",
-	# xprvA1DDS2qX9vCdRxSJmFA6AJPQPKS32S5hiUpzi9Xot9hzN4z1g6ip5oKJWXUeQDn2W48vaSWYVLaypC1JztUcrx23kucEtFzyU8t5Ay7NrDD
-        "xpub6ECZqYNQzHkveSWmsGh6XSL8wMGXRtoZ5hkbWXwRSVEyEsKADe34dbdnMob1ZjUpd4TD7no1isnnvpQq9DchFes5DnHJ7JupSntZsKr7VbQ"
-]
-
-# The CSV for the spend transaction
-# This option MUST NOT be changed after running revaultd for the first time. If you want to change it, please remove the following files:
-# - /path/to/your/data_dir/revaultd.sqlite3
-# - /path/to/your/data_dir/revaultd-watchonly-wallet-1
-unvault_csv = 18
+# The specifications of the Bitcoin Script that we are going to be tracking onchain, put here your own that you can
+# generate with the `mscompiler` tool (in `contrib/tools`).
+# These MUST NOT be changed after running revaultd for the first time, or you'll have to re-generate the database.
+# If you have to change it, be sure to remove the previous db at `/path/to/your/data_dir/network/revaultd.sqlite3`.
+[scripts_config]
+deposit_descriptor = "wsh(multi(1,xpub6BhQvtXJmw6hi2ALFeWMi9m7G8rGterJnMTNRqUm29uvB6dVTELvnEs7hfxyN3JM48FR2oh4t8chsvw7bRRRukkyhqp9WZD4oB9UvxAMpqC/*,xpub6BhQvtXJmw6hksh9rRRfdLjaWjQiNMZWtkM5ebn8QkAgh5na2Un6mCDABwkUmHhPCMYtsM9zHY5jxbQ86ayvjfY8XtavbovB6NcNy8KyQLa/*))#4s76hpqg"
```

Comment:
I would still include the xprivs here

Reply:
Ok i had to regenerate everything since i didn't save the xprivs :sob: . Done.